scripts/get_zeton_token.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
Let's include it also in the fullstack's `Makefile`

Reply:
Sure, [here it is](https://github.com/zetonteam/zeton_full_stack/commit/14ae578b82afaf45212e44c1f4722665fc50cba7) :grin: I accidentally pushed to `main` branch :shrug: branch protection feature is not enabled :stuck_out_tongue: